falcon/response.py

```diff
@@ -136,6 +145,82 @@ def set_stream(self, stream, stream_len):
         self.stream = stream
         self.stream_len = stream_len
 
+    def set_cookie(self, name, value, expires=None, max_age=None,
+                   domain=None, path=None, secure=True, httponly=True):
+        """Set a response cookie.
+
```

Comment:
What do you think about adding a note here talking about why this method should be used in lieu of set_header or append_header?